ChangeLog

```diff
@@ -1,5 +1,10 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+Bugfix
+   * Fix to allow test-ref-config.pl to build with Clang. Fixes #1628.
```

Comment:
There's something wrong with the changelog entry: #1628 says nothing about `test-ref-configs.pl`.
```suggestion
   * Allow building test suites with "clang -Wall -Wextra -Wunused -Werror". Fixes #1628.
```